src/stats.cc

```diff
@@ -55,61 +55,58 @@ value_t report_statistics(call_scope_t& args)
   assert(is_valid(statistics.earliest_post));
   assert(is_valid(statistics.latest_post));
 
-  out << format(_f("Time period: %1% to %2% (%3% days)")
```

Comment:
@jwiegley Can you thing of a reason why the outer `format` may have been significant?